bot/exts/fun/snakes/_utils.py

```diff
@@ -355,7 +355,7 @@ def frame_to_png_bytes(image: Image) -> io.BytesIO:
 
 
 log = get_logger(__name__)
-START_EMOJI = "\u2611"     # :ballot_box_with_check: - Start the game
+START_EMOJI = Emojis.check     # :ballot_box_with_check: - Start the game
 CANCEL_EMOJI = "\u274C"    # :x: - Cancel or leave the game
 ROLL_EMOJI = "\U0001F3B2"  # :game_die: - Roll the die!
 JOIN_EMOJI = "\U0001F64B"  # :raising_hand: - Join the game.
```

Comment:
These emoji constants previously had aligned comments but it's now misaligned. Perhaps consider either removing the extraneous space before `#` for `START_EMOJI` or re-align the comments?

Reply:
Comments have been addressed